mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -262,7 +262,7 @@ class MultiBroadcastConverter final
     if (outRank > inRank) {
       SmallVector<int64_t, 5> newShape = llvm::to_vector<5>(inShape);
       for (uint32_t i = inRank; i < outRank; i++) {
-        newShape.push_back(i);
+        newShape.push_back(1);
```

Comment:
This need backported?

Reply:
Not necessarily... (could do as well) because we wont hit the case unless we offload scalar literals from migraphx (which we currently dont) all others offloads has iso-rank for multibroadcasts.

Im fine with either.